src/quantity.rs

```diff
@@ -391,6 +391,19 @@ macro_rules! quantity {
             }
         }
 
+        impl<U, V> $quantity<U, V>
+        where
+            U: __system::Units<V> + ?Sized,
+            V: $crate::num::Num + $crate::Conversion<V> + $crate::ConstZero,
+        {
+            /// Constant representing the zero value.
+            pub const ZERO: Self = Self {
+                dimension: $crate::lib::marker::PhantomData,
+                units: $crate::lib::marker::PhantomData,
+                value: V::ZERO,
+            };
+        }
```

Comment:
Could this impl block be merged with one in `system.rs` and the `const` changed to a `const fn`? Would both optimize down to the same inlined code anyway?

```rust
const fn zero() -> Self
    where V: $crate::ConstZero
{
    ....
}
```

Reply:
> Could this impl block be merged with one in system.rs and the const changed to a const fn?

I will try, but I fear that going for a `const fn` will bring back the unstable trait bounds on `const fn` problem described in https://github.com/iliekturtles/uom/issues/26#issuecomment-614718476

> Would both optimize down to the same inlined code anyway?

That should definitely be the case as all evaluation happens at compile time with the resulting bits being "burned into" the executable.

Reply:
Indeed, a `const fn` seems out of reach on stable Rust for now. But I can move the impl block to `system.rs` next to the impl block for the `Zero` trait and actually turn it into an impl block for the `ConstZero` trait so that symmetry is preserved.